x-pack/legacy/plugins/apm/e2e/README.md

```diff
@@ -7,7 +7,7 @@
 ```shell
 $ git clone https://github.com/elastic/apm-integration-testing.git
 $ cd apm-integration-testing
-./scripts/compose.py start master --no-kibana --no-xpack-secure
+./scripts/compose.py start master --no-kibana --elasticsearch-port 9201 --apm-server-port 8201
```

Comment:
Enabling Security and ES and APM Server on non-default ports to avoid clashing with local development